convex/boards.ts

```
@@ -0,0 +1,280 @@
+    }
+
+    const board = await ctx.db.get(args.boardId);
+    if (!board) {
+      throw new Error("Board not found");
+    }
+
+    if (board.userId !== user._id) {
+      throw new Error("You don't have permission to update this board");
+    }
+
+    const updates: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)